tests/utils/manifestUtils.test.js

```diff
@@ -219,4 +220,114 @@ describe('manifestUtils tests', () => {
 
     expect(() => getRegionFromAppConfig('/test/project')).toThrow('Permission denied')
   })
-})
+
+  test('writeRegionToAppConfig: should warn when overwriting different region', () => {
+    const consoleWarnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {})
+    const configPath = '/test/project/app.config.yaml'
+    const existingConfig = {
+      application: {
+        runtimeManifest: {
+          database: {
+            region: 'us-west-2',
```

Comment:
This is an AWS "region", but in App Builder regions are labelled **amer**, **emea** and **apac**. It would be better to use these in the test.